src/chains/ethereum_1/helpers/generalModelHelpers.ts

```diff
@@ -0,0 +1,93 @@
+import moment from 'moment'
+import { EthereumDate, EthereumEntityName, EthereumAsset } from '../models/generalModels'
+import { isNullOrEmpty } from '../../../helpers'
+import { toWei } from './transactionHelpers'
+import {
+  toEthUnit,
+  isValidEthereumAddress,
+  isValidEthereumPublicKey,
+  isValidEthereumTxData,
+  isValidEthereumPrivateKey,
+} from './cryptoModelHelpers'
+
+export function isValidEthereumDateString(str: string): str is EthereumDate {
+  if (isNullOrEmpty(str)) return false
+  return str.match(/^\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}\.\d{2}\d+$/i) !== null
+}
+
+export function isValidEthereumAsset(symbol: string): boolean {
```

Comment:
This function is confusingly named and probably not what is expected. isValidEthereumAsset should be used to test an asset param assigned for an ETH contract/action.
This function as written does something different - it checks whether a string is a valid ETH asset unit type. I would change the param from 'symbol' to 'unitName' and rename this function isValidEthereumAssetUnit.
Then I would add a new function isValidEthereumAsset that checks a value to see if it is a well-formed asset value - which is just a positive number. Right?

Reply:
Must be clear now